crates/sui-json-rpc/src/lib.rs

```diff
@@ -86,12 +87,15 @@ impl JsonRpcServerBuilder {
         };
         info!(?acl);
 
+        let client_type: HeaderName = HeaderName::from_static("client-type");
+        let client_api_version: HeaderName = HeaderName::from_static("client-api-version");
+
```

Comment:
@patrickkuo do you know if this is necessary?

Reply:
yes we probably need this when calling from a different host